account_analytic_distribution_manual/models/account_analytic_distribution_manual.py

```diff
@@ -10,6 +11,15 @@ class AccountAnalyticDistributionManual(models.Model):
 
     name = fields.Char(required=True)
     active = fields.Boolean(default=True)
+    analytic_distribution_import = fields.Json(
+        compute="_compute_analytic_distribution_import",
+        inverse="_inverse_analytic_distribution_import",
+        readonly=False,
+        string="Analytic distribution (importable)",
+        help="""Defining this field, it will set the analytical distribution in JSON
+        format, but using the analytic accounts names as keys of the dictionary, so it
+        eases the human input.""",
```

Comment:
I don't think you should use the triple quote, as this introduces the line feeds and spaces in the text and translations:

```suggestion
        help="Defining this field, it will set the analytical distribution in JSON "
        "format, but using the analytic accounts names as keys of the dictionary, so it "
        "eases the human input.",
```